asv_bench/benchmarks/benchmarks_hindcast.py

```diff
@@ -0,0 +1,166 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+
+
+import dask
+import numpy as np
+import xarray as xr
+
+from climpred.bootstrap import bootstrap_hindcast
+# from climpred.constants import HINDCAST_COMPARISONS
+from climpred.prediction import compute_hindcast
+
+from . import parameterized, randn, requires_dask
+
+HINDCAST_COMPARISONS = ['m2r']  # e2r and probabilistic dont match
```

Comment:
Should be `m2o` now. Can you edit the comment to make it a little more clear what you mean? Also I guess this would disappear now that you will import it with the circular imports fix.